stack/indexer/deb/debian/postinst

```diff
@@ -14,12 +14,13 @@ export USER=${NAME}
 export GROUP=${NAME}
 export CONFIG_DIR=/etc/${NAME}
 export INSTALLATION_DIR=/usr/share/${NAME}
+export BACKUP_DIR="${CONFIG_DIR}/backup"
```

Comment:
We have noticed that this directory is used to back up the passwords by `wazuh-passwords-tool.sh`, please rename it to `upgrade_backup`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/1998/commits/39bdda9c57545dc2e1d87fdc0aa606a46158c88a.